### PR TITLE
feature/share-panel

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
#### Summary
`SharePanel` component + `ShareModalTrigger` to implement it as the modal window.

#### Screenshots
image

### CODE REVIEW

**Comment by Partysun** [comment]
@DmitriVanGuard We have 2 buttons in our Figma for this modal.
image
image
Can we add it as part of this PR?

File: src/components/SharePanel/ModalSharePanel.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import { Modal } from 'semantic-ui-react'
+import SharePanel from './SharePanel'
+
+class ModalSharePanel extends Component {
+  state = { isOpen: false }
+
+  closeModal = () => {
```
Comment (Partysun) [inline src/components/SharePanel/ModalSharePanel.js]: Can we make one toggle fnc here? @DmitriVanGuard

**Comment by DmitriVanGuard** [comment]
@Partysun Sure. I will add these

File: src/components/SharePanel/SharePanel.module.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
+@import "~@santiment-network/ui/variables.scss";
+@import "~@santiment-network/ui/mixins.scss";
+
+.wrapper {
+    min-width: 350px;
```
Comment (Partysun) [inline src/components/SharePanel/SharePanel.module.scss]: @DmitriVanGuard Default value for mobile devices, and mixin breakpoint value for table and desktop.
For mobile devices, we need to have auto full width.

**Pull request merged**